docs/management/admin/host-isolation-exceptions.asciidoc

```diff
@@ -14,20 +16,28 @@ IMPORTANT: Each host isolation exception IP address should be a highly trusted a
 .. `Name your host isolation exceptions`: Enter a name to identify the host isolation exception.
 .. `Description`: Enter a description to provide more information on the host isolation exception (optional).
 .. `Enter IP Address`: Enter the IP address for which you want to allow communication with an isolated host. This must be an IPv4 address, with optional CIDR notation (for example, `0.0.0.0` or `1.0.0.0/24`, respectively).
-. Click **Add Host isolation exception**.
+. Select an option in the *Assignment* section to assign the host isolation exception to a specific integration policy:
```

Comment:
I know you mentioned it above, but may also want to add a platinum/enterprise note here

Reply:
There's a note below, which we can move up to the entire step.

Reply:
I think it makes sense to just mention that the entire feature is platinum and we can remove the other references.  Thoughts @caitlinbetz ?

Reply:
oh yes that makes sense. Sorry, I think I meant to comment this on the event filters page. This looks good to me.